microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -83,7 +83,12 @@ On the **Properties page**, enter the following information to specify propertie
 | Support contact details | The URL to your support webpage, or the email address to contact your support team.  |  
 | Mature content | Checkbox to specify if your extension includes mature content.  Extension rating helps determine the appropriate age group of the target audience of your extension.  To help determine if your extension has mature content, go to [Microsoft Edge add-ons catalog developer policies][MicrosoftEdgeAddonsCatalogDeveloperPolicies].  |  
 
-Select **Save** to continue to the **Store listings** section.  
+
+Select **Save** to continue to the **Store listings** section.
+
+> [!Important]
+> Your developer/organization name, website URL and the support contact details are displayed to the users on the Edge add-ons store.
```

Comment:
Does the name come from the manifest file? Can we include that clarification?
 
Your developer/organization name included in the manifest file, website URL...

Reply:
It comes from account details which are submitted at the time of developer registration.

Reply:
Can we update to:

> Your developer/organization name, website URL, and support contact details that were submitted during registration are displayed to users on the Microsoft Edge add-ons store.

Reply:
done